### PR TITLE
[Remove Stream/Optional usage] LPS-178247 remove_stream_and_optional_in_commerce_qualifier_service

### PR DESCRIPTION
**Pull request by jeffwu0724**
Hi Tina, Dante,

Resent: https://github.com/liferay-core-infra/liferay-portal/pull/3837
This is for ticket https://issues.liferay.com/browse/LPS-178247
Thanks for checking!

### CODE REVIEW

File: modules/apps/commerce/commerce-qualifier-service/src/main/java/com/liferay/commerce/qualifier/internal/metadata/CommerceQualifierMetadataRegistryImpl.java [hunk 1]
```diff
@@ -49,19 +47,17 @@ public CommerceQualifierMetadata getCommerceQualifierMetadata(
...
 
-		Collection<CommerceQualifierMetadata> commerceQualifierMetadatas =
-			_serviceTrackerMap.values();
+		for (CommerceQualifierMetadata commerceQualifierMetadata :
+				_serviceTrackerMap.values()) {
 
-		Stream<CommerceQualifierMetadata> stream =
-			commerceQualifierMetadatas.stream();
+			if (Objects.equals(
+					className, commerceQualifierMetadata.getModelClassName())) {
```
Comment (jeffwu0724) [inline modules/apps/commerce/commerce-qualifier-service/src/main/java/com/liferay/commerce/qualifier/internal/metadata/CommerceQualifierMetadataRegistryImpl.java:54]: put commerceQualifierMetadata.getModelClassName() at second

**Comment by zhangyan0701** [comment]
ci:test:commerce

**Comment by catarineaguiar** [comment]
no regressions found.

**Comment by dantewang** [comment]
ci:forward:force

cc @shuyangzhou @tinatian

**Pull request closed without merging**